packages/twenty-front/src/modules/workflow/components/FormFieldInput.tsx

```diff
@@ -0,0 +1,26 @@
+import VariableTagInput from '@/workflow/search-variables/components/VariableTagInput';
+
+type FormFieldInputProps = {
+  recordFieldInputdId: string;
+  label: string;
+  value: string;
+  onChange: (value: string) => void;
+  isReadOnly?: boolean;
+};
+
+export const FormFieldInput = ({
```

Comment:
That component is not workflow specific. Could you move it next to `FieldInput.tsx` 🙏